### PR TITLE
Added logic  and and view to display user files

### LINKED ISSUE #93: storing user uploaded files in user model
**Actual Behaviour**

currently, the user cannot see the uploaded files by him

**Expected Behaviour**

Each user after logging in can able to see their files listed

**Would you like to work on the issue?**
I want to work on this issue

### PR DESCRIPTION
**Pull request by mark-nakachon**
### Checklist:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #93

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -166,6 +167,23 @@ app.post('/login', (req, res, done) => {
 		successRedirect: '/'
 	})(req, res, done);
 });
+//get the uploaded files
+app.get('/uploaded-files', (req, res) => {
+	File.find({
+		user: req.user.id
+	}).then(files => {
+		console.log(files);
```
Comment (rossoskull) [inline server/index.js]: Not required

File: views/pages/files.ejs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="container mt-5 p-5">
+        <% files.forEach(function(file){ %>
+        <div class=" p-2 d-flex justify-content-between  border border-dark col-md-12 m-2">
+            <p class="text-primary"><%= file.file %></p>
+            <button class="btn btn-success">Download</button>
+        </div>
+        <%}) %>
+    </div>
+</body>
+<script>
```
Comment (rossoskull) [inline views/pages/files.ejs]: Unnecessary script tag

**Comment by mark-nakachon** [comment]
@rossoskull I made changes

**Comment by mark-nakachon** [comment]
Fixed comflicts

File: views/pages/files.ejs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Upload File</title>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet"
```
Comment (aashutoshrathi) [inline views/pages/files.ejs]: These css imports are already in header.ejs. Right?

File: views/pages/files.ejs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<body>
+    <div class="container mt-5 p-5">
+        <% files.forEach(function(file){ %>
+        <div class=" p-2 d-flex justify-content-between  border border-dark col-md-12 m-2">
+            <p class="text-primary"><%= file.file %></p>
+            <button class="btn btn-success">Download</button>
+        </div>
+        <%}) %>
+    </div>
+</body>
```
Comment (aashutoshrathi) [inline views/pages/files.ejs]: Add newline at the end

**Comment by mark-nakachon** [comment]
Screenshot from 2019-05-17 19-15-30

File: views/pages/files.ejs (hunk 3 above)
Comment (mark-nakachon) [reply views/pages/files.ejs]: Fixed imports

File: views/pages/files.ejs (hunk 4 above)
Comment (mark-nakachon) [reply views/pages/files.ejs]: Added new line

**Comment by aashutoshrathi** [comment]
Resolve conflicts

**Comment by aashutoshrathi** [comment]
Also, one request that please write `Fixes #93` instead of `Fixes #{93}` in PR Description

**Pull request merged**